api_eucovidcert.yaml

```diff
@@ -0,0 +1,65 @@
+swagger: "2.0"
+info:
+  version: 0.0.1
+  title: EU Covid Certificate service for IO App
+basePath: /eucovidcert/api/v1
+schemes:
+  - https
+security:
+  - Bearer: []
+paths:
+  /certificate:
+    post:
+      operationId: getCertificate
+      summary: Retrieve a certificate for a given Citizen
+      description: |-
+        Given a Citizen's fiscal code and an OTP (the auth code previously sent via IO Message), a Certificate is returned with ID, encoded QR-code, expiration date and a markdown text with all meaningful information to be shown to Citizens.
+      parameters:
+        - name: accessData
+          in: body
+          schema:
+            $ref: "#/definitions/GetCertificateParams"
+      responses:
+        '200':
+          description: |-
+            A Certificate exists and it's found for the given access data. It is retrieved regardless of it's expired or its current status
+          schema:
+            $ref: "#/definitions/Certificate"
+        '400':
+          description: Payload has bad format
+        '403':
+          description: Access data provided are invalid or no Certificate has been emitted for the given Citizen
+        '410':
+          description: Endpoint no longer avaible
+        '500':
+          description: Generic server error
+        '504':
+          description: Gateway Timeout
+          
+definitions:
+  GetCertificateParams:
+    type: object
+    properties:
+      auth_code:
+        type: string
+        description: OTP previously sent to the Citizen
+    required:
+      - auth_code
+  Certificate:
+    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-eucovidcerts/master/openapi/index.yaml#/definitions/Certificate"
```

Comment:
We should not refer to `master` but pin to a specific version.

Reply:
I know, I did it right now because it's too early to have a fixed version io fn-eucovidcert api.

Reply:
I can stand that.